tpm2tools/seal_test.go

```diff
@@ -0,0 +1,238 @@
+package tpm2tools
+
+import (
+	"bytes"
+	"crypto/sha256"
+	"fmt"
+	"testing"
+
+	"github.com/google/go-tpm/tpm2"
+	"github.com/google/go-tpm/tpmutil"
+
+	"github.com/google/go-tpm-tools/internal"
+)
+
+func TestSeal(t *testing.T) {
+	rwc := internal.GetTPM(t)
+	defer rwc.Close()
+
+	key, err := StorageRootKeyRSA(rwc)
+	if err != nil {
+		t.Fatalf("can't create srk from template: %v", err)
+	}
+
+	secret := []byte("test")
+	pcrList := []int{1, 2, 3, 4, 5, 6, 7}
```

Comment:
Two things:
a) This should be a var defined above as it's used multiple times
b) Why are we extending 7 different PCRs here? Why not just extend a single PCR between our 2 unseal attempts?

Reply:
a) Moving this to be a file constant makes the tests seem more implicit, which I'd like to avoid. 
b) Using more than one seems like a negligibly more thorough test. Easy to change if you want it to be one.

Reply:
changed.